codebuild_specs/migration_tests_v10.yml

```diff
@@ -20,7 +20,7 @@ env:
     APPLE_TEAM_ID: '2QLEWNDK6K'
     APPLE_KEY_ID: '2QLZXKYJ8J'
     # mock value, Cognito validates the private key, this is an invalidated key.
-    APPLE_PRIVATE_KEY: '----BEGIN PRIVATE KEY-----MIGTAgEAMBMGByqGSM49AgEGCCqGSM49AwEHBHkwdwIBAQQgIltgNsTgTfSzUadYiCS0VYtDDMFln/J8i1yJsSIw5g+gCgYIKoZIzj0DAQehRANCAASI8E0L/DhR/mIfTT07v3VwQu6q8I76lgn7kFhT0HvWoLuHKGQFcFkXXCgztgBrprzd419mUChAnKE6y89bWcNw----END PRIVATE KEY----'
+    APPLE_PRIVATE_KEY_2: '----BEGIN PRIVATE KEY-----MIGTAgEAMBMGByqGSM49AgEGCCqGSM49AwEHBHkwdwIBAQQgIltgNsTgTfSzUadYiCS0VYtDDMFln/J8i1yJsSIw5g+gCgYIKoZIzj0DAQehRANCAASI8E0L/DhR/mIfTT07v3VwQu6q8I76lgn7kFhT0HvWoLuHKGQFcFkXXCgztgBrprzd419mUChAnKE6y89bWcNw----END PRIVATE KEY----'
```

Comment:
On the codebuild side, we were able to fix this, but since both codebuild & circleci are going to be up in the meantime we must use the same variable names and revert to the "_2" hack